docs/source/algorithms/RotateSource-v1.rst

```diff
@@ -0,0 +1,54 @@
+
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm corrects the source's position by rotating it around an axis centered at the sample.
+The rotation axis is perpendicular to the plane determined by the beam direction and the *up* direction.
```

Comment:
Ok. We just want to be explicit about how the handedness interplays with the angle sign here.